bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh

```diff
@@ -102,7 +102,7 @@ function retry-command() {
     sleep 15
   done
   if [ ${kubeadm_return} -ne 0 ]; then
-    log::error_exit "too many errors, exiting"
+    log::error_exit "too many errors, exiting" "${kubeadm_return}"
```

Comment:
not sure if we want to set a different exit code or pass through the kubeadm_return

/cc @randomvariable

Reply:
I think pass through is fine.